webapp/src/Controller/API/UserController.php

```diff
@@ -292,6 +294,90 @@ public function singleAction(Request $request, string $id)
         return parent::performSingleAction($request, $id);
     }
 
+    /**
+     * Add a new user
+     *
+     * @Rest\Post()
+     * @Security("is_granted('ROLE_ADMIN') or is_granted('ROLE_API_WRITER')")
```

Comment:
Is admin not also a API WRITER?

Reply:
HEY you are right. So we need only api writer here.